styles.css

```diff
@@ -1,267 +1,119 @@
 
-/* TYPOGRAPHY */
-header                                              {font-family: "expressway", sans-serif; font-style: normal}
-#slivers h2, #slideshow                             {font-family: Georgia}
-#slivers h2                                         {font-size: 22px; line-height: 24px; font-weight: normal}
-.navbar-nav, footer                                 {font-size: 11px}
-.navbar-nav a                                       {font-weight: bold}
-footer a:focus, footer a:hover                      {text-decoration: none}
-#slideshow blockquote                               {font-size: 16px}
-#slideshow p.more                                   {font-size: 14px; font-style: italic}
+/* BACKGROUNDS */
+body                                                {background-image: url(images/topbluefade.gif); background-repeat: repeat-x}
+.slivers                                            {background-image: url(images/herobackground.gif)}
+@media (min-width: 768px) {header.container         {background-image: url(images/rails.png); background-repeat: no-repeat; background-position: 22px 0px}}
 
-footer                                              {text-align: center}
-#announce                                           {font-size: 11px; line-height: 13px}
-#slivers p                                          {font-size: 10px}
-h1                                                  {font-size: 38px; line-height: 30px; font-weight: 700}
-header h2                                           {font-size: 21px; line-height: 24px; font-weight: 400}
-@media (max-width: 768px) {
-  h1                                                {font-size: 31px}
-  header h2                                         {font-size: 18px}
-}
+/* COLORS */
+a                                                   {color: #369}
+a:hover, a:focus                                    {background-color: #f4dda6}
+mark                                                {background-color: #ffc}
+footer                                              {color: #666}
+#menu a, code, pre, footer a                        {color: #000}
+#menu a:focus, #menu a:hover                        {color: #fff; background-color: #333}
+.navbar-default, code, pre                          {background-color: transparent}
+.navbar                                             {border-color: transparent}
+h4                                                  {color: #999}
+.framed img, .col-sm-4 img                          {background-color: #fff; border-color: #ccc}
+h5                                                  {color: #900}
+cite                                                {color: #a69e8a}
+#announce                                           {background-color: #981a21}
+#announce a, .inverted a:hover, .inverted a:focus   {color: #fff}
+.slivers                                            {border-top-color: #f6e69f; border-bottom-color: #f6e69f}
```

Comment:
@rafaelfranca Let me see if I can convince you whether the style I applied is preferable… if I don't, I'll switch to multiline. :grin: 

[In this gist](https://gist.github.com/claudiob/8e4407d2109fa73ab9bb) I show the same CSS from this PR, both in the current style (inline) and in the style you suggest (multi-line).

What I like about the style that I applied is that you can see all the CSS rules in context. 

Take for instance `font-size`: you can understand with a single glance what are the "ranges" of font sizes in this web site. In case you want to add a new element in the future, you can clearly see where its font size will fit into (e.g., you can simply set its class to `footnote` if you want 12px, without having to add an extra-class at the bottom of the CSS, just because you have no clue what all the font sizes represent). 

Same thing happens with colors: you have an idea of the palette already used, so if you decide to assign a meaning to a new color (e.g. if "red" means "danger" for you), then you can immediately see if some elements are using similar colors, and avoid confusions.

---

Having less lines of CSS is beneficial not just for readability, but also for maintainability.

For instance, say that in the future commit you decide to change the color of `<h4>` elements from `#999` to `#555`. 
With the inline style, `git blame` would simply show you a one-line change, such as:

```git
dcf2aa "Changed color of H4"      46) h4 {color: #555}
```

With multi-line style, you would get something like this:

```git
57fe5f "Make rubyonrails.org responsive" 138) h4 {
dcf2aa "Changed color of H4"             139)   color: #999
57fe5f "Make rubyonrails.org responsive" 140) }
```
which (in my opinion) is harder to parse than an atomic change to the full line that sets the color of `<h4>`.


---

Finally, some lines include multiple CSS properties, such as:

```css
#menu a:focus, #menu a:hover {color: #fff; background-color: #333}
```

If you look carefully, this lines group _related properties_ together. For instance, the line above sets the color to white and the background to dark gray. Keeping these properties together prevents people from just changing one and not realizing that it's tightly coupled to the other one. If you suddenly decide to change the `color` from white to black, it's useful to see right there that you are going to have problems unless you also change the background from dark gray to something lighter.

The same applies to layout:

```css
#menu li {margin-top: 10px; margin-bottom: 13px}
```
This line groups two properties that you can play around with if you want to keep everything below the menu in the same position but shift the menu items some pixels to the top or the bottom.

---

I've been using this style for years, and it's been really helpful to maintain short CSS files for a long period. I understand that, in this specific case, we are talking about a project shared among many developers, and about a CSS file that is not very short because it's meant to maintain the style **exactly** as it was before this PR.

However, I think this style of CSS can also help us make the site more coherent by gradually removing CSS rules that are "very similar but not exactly the same". An example would be to make all the vertical spacing (padding/margin) more uniform among the various selectors.

---

Ok… this feels like a rant :space_invader: and it might not be easy to understand, so just let me know, if you still feel better with the multi-line style, I'll just switch to that! Thanks for reading :ribbon: